outputs.tf

```diff
@@ -24,7 +24,7 @@ output "attributes" {
 }
 
 output "tags" {
-  value       = "${local.tags}"
+  value       = "${local.tags[var.user_tags_only ? "user" : "all"]}"
   description = "Normalized Tag map"
```

Comment:
Also, you've mentioned tags as normalized . In fact there's no normalization... keys are untouched and basically everything is outputted exactly as it was passed via variable...

Reply:
These are the normalized fields: 
https://github.com/stackfeed/terraform-null-label/blob/c0d03bea486fc175d6e555737bd98acb4235fb29/main.tf#L3-L7

They are lowercased. Attributes are joined with a delimiter.

Reply:
yup, right i've missed that